agr_java_core/src/main/java/org/alliancegenome/neo4j/view/ParalogView.java

```diff
@@ -0,0 +1,87 @@
+package org.alliancegenome.neo4j.view;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonView;
+import lombok.Getter;
+import lombok.Setter;
+import org.alliancegenome.neo4j.entity.node.Gene;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+@Setter
+@Getter
+@Schema(name="OrthologView", description="POJO that represents the Ortholog view")
+@JsonPropertyOrder({"gene", "homologGene", "length", "similarity", "identity", "rank", "predictionMethodsMatched", "predictionMethodsNotMatched", "predictionMethodsNotCalled", "methodCount", "totalMethodCount"})
+public class ParalogView implements Serializable {
```

Comment:
Can this have a different name than View so that its doesn't get confused with actual views call it summary or something